api/.env-example

```diff
@@ -2,13 +2,13 @@
 PORT=3001
 
 # Assuming that you started your MySQL container with the below command:
-# docker run --name mysql -e MYSQL_ROOT_PASSWORD=mysql-root-password -e MYSQL_DATABASE=my-database -d -p 3306:3306 mysql:latest
+# docker run --name mysql -e MYSQL_ROOT_PASSWORD=my-secret-pw -e MYSQL_DATABASE=my-database -d -p 3306:3306 mysql:latest
 DB_CLIENT=mysql2
 DB_HOST=localhost
 DB_PORT=3306
 DB_USER=root
-DB_PASSWORD=mysql-root-password
-DB_DATABASE_NAME=meal-sharing
+DB_PASSWORD=my-secret-pw
+DB_DATABASE_NAME=meal_sharing_website
 
 # The database you use for local development most likely doesn't use SSL (encryption)
 # but the one you deploy on Render.com does.
```

Comment:
This file was supposed to be renamed from `.env-example` to `.env`. then `.env` is listed in your `.gitignore` here:
https://github.com/Nusrath-Jahan/meal-sharing/blob/main/api/.gitignore

This means your code repository never commits your passwords into the version control software `git`. Isn't that clever.

Here it seems you have shared your passwords with the world

Reply:
Hello Rasmus,
At First I apologize for my homework that I copied from someone. 😞😞😞
Now I did it myself. I am really sorry for that.
I already have the .evn file that I attached to the .gitignore file. That's why I deleted the .env-example file.
And my database password, I don't use passwords anymore.
Thank you for your valuable suggestions.